src/styles/components/form-elements/styles.less

```diff
@@ -34,4 +34,10 @@
     left: 100%;
     position: absolute;
   }
+
+  .form-remove {
+    position: absolute;
```

Comment:
@mlunoe This is to position the "remove" icon
![](https://cl.ly/3R312X2x340L/Screen%20Shot%202016-11-16%20at%204.55.13%20PM.png) and I do think absolute positioning makes sense here, personally.